torch/_dynamo/skipfiles.py

```diff
@@ -203,31 +186,35 @@ def _module_dir(m: types.ModuleType):
         "torch.distributed.tensor.parallel.style",
     }
 
-# Include optimizer code for tracing
-FILE_INLINELIST |= {
-    str(obj.__module__) for obj in torch.optim.__dict__.values() if inspect.isclass(obj)
-}
 
-# TODO: consolidate SUBMODULE_INLINELIST and FILE_INLINELIST into one list
-# Force inline functions under these modules, even the modules is in *_SKIPLIST.
-SUBMODULE_INLINELIST = {
+# Force inline functions under these modules, even they are in *_SKIPLIST.
+# We are using python module name instead of file or directory object to avoid circular dependency.
+# Please keep this sorted alphabetically.
```

Comment:
To keep it sorted alphabetically, you should make this a list rather than a set, and then assert that the list is in sorted order. Or write a lint rule for it, but the assert is easier IMO

Reply:
This should be a set rather than list. Keeping this sorted is to help developers easily find if a module has been listed here, so probably I have to add a lint rule.